Lib/ufo2ft/featureWriter/__init__.py

```diff
@@ -1,19 +1,2 @@
-from __future__ import (
-    print_function, division, absolute_import, unicode_literals)
-
-
-def liststr(glyphs):
-    """Return string representation of a list of glyph names."""
-    return "[%s]" % " ".join(glyphs)
-
-
-class BaseFeatureWriter(object):
-    """Abstract features writer"""
-
-    def __init__(self, font, features=()):
-        self.font = font
-        self.features = features
-
-    def write(self, **kwargs):
-        """Write features and class definitions"""
-        raise NotImplementedError
+from ufo2ft.featureWriter.kernFeatureWriter import KernFeatureWriter
+from ufo2ft.featureWriter.markFeatureWriter import MarkFeatureWriter
```

Comment:
I would also export `BaseFeatureWriter` class in the `ufo2ft.featureWriter` top-level module